Lesson20/Les20-fercan/script.js

```diff
@@ -0,0 +1,112 @@
+// 1. Create a Person class with name, age, and gender properties, and a greet() method that logs a greeting message to the console.
+// Then create a Student class that extends the Person class and adds a major property. Override the greet() method
+// in the Student class to log a different greeting message to the console. 
+//Create 2 variables, one of type Person and one of type Student and log them to console.
+
+
+class Person {
+    constructor (name, age, gender) {
+        this._name= name;
+        this._age= age;
+        this._gender = gender;
+
+    }
+    greet () {
+    return 'Welcome to my world...My name is ${this._name}';
+}
+
+}
+const person1 = new Person('adam', 34, 'female');
+console.log('Exercise 1: ');
+console.log(person1.greet());
+
```

Comment:
For this exercise, create one more class (Student). It should extend the class Person and have one more property (major). The greet() method of the Student class should print a different message.